networks/local/localnode/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.7
+FROM alpine:3.14
```

Comment:
We could also wait until upstream cometbft upgrades this line and we merge latest changes from upstream here.